test/solve-lib/DeFi/FlashLiquidityLib.sol

```diff
@@ -20,14 +20,18 @@ contract FlashLiquidityLib {
     CallBreaker public callbreaker;
     uint256 _tipWei = 33;
 
+    uint256 public balanceOfWeth;
+    uint256 public balanceOfDai;
```

Comment:
This is unused

Reply:
Yes I have this here because we might add return price calculations later